variables.tf

```diff
@@ -156,7 +156,7 @@ variable "maintenance_window" {
 
 variable "skip_final_snapshot" {
   description = "Determines whether a final DB snapshot is created before the DB instance is deleted. If true is specified, no DBSnapshot is created. If false is specified, a DB snapshot is created before the DB instance is deleted, using the value from final_snapshot_identifier"
-  default     = true
+  default     = false
```

Comment:
:+1: for this! It would be in line with the Terraform resource defaults:
https://www.terraform.io/docs/providers/aws/r/db_instance.html#skip_final_snapshot

Also, it makes sense that the `final_snapshot_identifier` input is declared as required for the module in the registry:
https://registry.terraform.io/modules/terraform-aws-modules/rds/aws/2.14.0?tab=inputs#required-inputs